modules/file-system/pre-existing-network-storage/README.md

```diff
@@ -24,6 +24,22 @@ This creates a pre-existing-network-storage module in terraform at the
 provided IP in `server_ip` of type nfs that will be mounted at `/home`. Note
 that the `server_ip` must be known before deployment.
 
+### Mounting
+
+For some `fs_type`, this module will provide `client_install_runner` and
```

Comment:
nit: s/For some `fs_type`/For the `fs_type` listed below,/

Reply:
also, what will happen for the unsuported ones? (meaning, it will still populate the data structure, but will not provide the mounters, which is fine, as long as these are part of the image...)

Reply:
In the case it is unsupported there will still be a runner  added that just prints something like "Skipping, not supported". See the first and last lines of [this section](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/54270c169a6edbea50adbb50a1e919b9fc5d8cad/modules/file-system/pre-existing-network-storage/outputs.tf#L66-L75).

Reply:
The wording from your original substitution suggestion has been incorporated. Leaving comment open in case there is more discussion here.

Reply:
yes, but it will still populate the data structure, is that structure no longer being added to instance metadata?

Reply:
It is still being added to network storage. As it stand now it will always emit a runner, they will just be more or less a no-op if it is unsupported. 

Another valid option would be to set runners to `null` in the unsupported case (although it is not set up that way now). Would this be your preference? In one case we silently emit null, in the other case we emit a runner even when unsupported, but the runner logs that it is unsupported. 

`Null` maybe seems more appropriate but I could go either way and don't want to change unless that is also what you want.

Reply:
As discussed offline, we will leave as is: populate a runner with a warning.